benches/helpers.rs

```diff
@@ -151,11 +151,11 @@ pub async fn ws_server(handle: tokio::runtime::Handle) -> (String, jsonrpsee::se
 			SUB_METHOD_NAME,
 			UNSUB_METHOD_NAME,
 			|_params, pending, _ctx| async move {
-				let sink = pending.accept().await?;
+				let sink = pending.accept().await.unwrap();
 				let msg = SubscriptionMessage::from("Hello");
-				sink.send(msg).await?;
+				sink.send(msg).await.unwrap();
```

Comment:
Nit; is there any harm in using `?` here?

Reply:
no :)

I didn't change it after playing around with different approaches.